papers/gottumukkala/myst.yml

```diff
@@ -38,11 +38,3 @@ project:
         - Writing-original draft
 
   subject: Biology
-  venue: The Morganton Scientific
```

Comment:
A few of the `myst.yml` files are redefining these fields, which are already defined in the parent `yml`.